src/commands/configure/docker-source.ts

```diff
@@ -0,0 +1,27 @@
+import Command from '../../base'
+import CustomConfig from "../../libs/customConfig"
+
+export default class Docker_Source extends Command
+{
+  static description = 'Set or show docker source directory'
+
+  static args = [
+    {
+      name: 'key',
+      required: false,
+      description: 'Set the Docker source directory'
+    }
+  ]
+
+  static flags = {
+    ...Command.flags
+  }
+    
+  async run() {
+    if (this.args.command !== undefined) {
```

Comment:
Should it be this.args.key according to your args definition?
Also, I would suggest to name it as "value" instead of "key".